benchmark/realnest/12_map_two_list_values.benchmark

```diff
@@ -0,0 +1,19 @@
+# name: benchmark/realnest/12_map_two_list_values.benchmark
+# description: Map two list values
+# group: [realnest]
+
+name map_two_list_values
+group real_nest
+
+require json
+
+require httpfs
+
+cache real_nest.duckdb
+
+load benchmark/realnest/load.sql
+
+run
+SELECT map([gh_issue.issue.user.login],[gh_pull.pull_request.body]) 
```

Comment:
looking at this one again, I'm not sure what we are benchmarking. In this case 
`select [gh_issue.issue.user.login] as lst_login, [gh_pull.pull_request.body] as lst_body FROM gh_pull, gh_issue WHERE gh_issue.issue.user = gh_pull.pull_request.head.repo.owner
  GROUP BY ALL limit 5;` returns the following
```
┌────────────────────┬─────────────────────────────────────────────────────────┐
│     lst_login      │                        lst_body                         │
│     varchar[]      │                        varchar[]                        │
├────────────────────┼─────────────────────────────────────────────────────────┤
│ [matsumo0922]      │ [# Related links\r\n- N/A\r\n\r\n# How?\r\n- バージョ…  │
│ [TiwariRahul16]    │ [## PR Description 📜\r\n\r\nAdded [New game] Rock_pa…  │
│ [LashaJini]        │ [NULL]                                                  │
│ [Thraximundar2772] │ [NULL]                                                  │
│ [SimVac]           │ [close #42 ]                                            │
└────────────────────┴─────────────────────────────────────────────────────────┘
```

This basically means the benchmarked query is calling map on two lists of length 1 over and over again. In addition, we are benchmarking joining on useres again, which is done pretty heavily in `benchmark/realnest/13_multi_join_nested_data_with_filtering.benchmark`

I think what would be nice is to find a dataset with on table that has two list columns have of the same length (or we can make them the same length with `list_resize`) and then we call `map(list_1, list_2)`. Then map is called on two lists with. multiple different list lengths. 

@taniabogatsch what do you think?

Reply:
Yes, that makes a lot of sense. This is also why we resized the lists in one of the views?